src/cases/containers/allocate-role-container/allocate-role-container.component.spec.ts

```diff
@@ -0,0 +1,38 @@
+import { CUSTOM_ELEMENTS_SCHEMA } from '@angular/core';
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
+import { RouterTestingModule } from '@angular/router/testing';
+
+import { AllocateRoleContainerComponent } from '..';
+
+describe('AllocateRoleContainerComponent', () => {
+  let component: AllocateRoleContainerComponent;
+  let fixture: ComponentFixture<AllocateRoleContainerComponent>;
+
+  beforeEach(async(() => {
+    TestBed.configureTestingModule({
+      schemas: [CUSTOM_ELEMENTS_SCHEMA],
+      imports: [
+        RouterTestingModule
+      ],
+      declarations: [AllocateRoleContainerComponent],
+    })
+      .compileComponents();
+  }));
+
+  beforeEach(() => {
+    fixture = TestBed.createComponent(AllocateRoleContainerComponent);
+    component = fixture.componentInstance;
+
+    fixture.detectChanges();
+
+  });
+
+  describe('entering page', () => {
+
+    it('should create', () => {
```

Comment:
This is not testing anything.

Reply:
You're right. If I change this to toBeDefined() would that be suitable? There's not much to test but feel more comfortable creating a container with initial tests in place

Reply:
You could test if the activated route snapshot has populated the correct data

Reply:
I've updated with some actual tests now